### ISSUE TITLE
impossible to insert stack in crate attached to sailboat

### ISSUE THREAD

**Issue by Martel732** [anegostudios/VintageStory-Issues#8478]
### Game Version

V1.21

### Platform

Windows

### Modded

Modded

### SP/MP

Singleplayer

### Description

it is currently impossible to insert a stack of items in a crate if it is attached to a sailboat.
 making crate attached to sailboat virtually useless as it take forever to fill them (64*16 clicks to get the same load as a chest and an additional 64*4 clicks to fill the crate) 

### How to reproduce

1 make a sailboat
2 make a crate
3 attach crate to sailboat
4 select a stack of item in hotbar
5 try to insert stack of item in crate by CTRL+shift+right clicking the crate

### Logs (Required)

[server-debug.log](https://github.com/user-attachments/files/25724126/server-debug.log)
[server-main.log](https://github.com/user-attachments/files/25724121/server-main.log)
[server-worldgen.log](https://github.com/user-attachments/files/25724123/server-worldgen.log)
[client-audit.log](https://github.com/user-attachments/files/25724122/client-audit.log)
[client-chat.log](https://github.com/user-attachments/files/25724120/client-chat.log)
[client-debug.log](https://github.com/user-attachments/files/25724128/client-debug.log)
[client-main.log](https://github.com/user-attachments/files/25724129/client-main.log)
[server-audit.log](https://github.com/user-attachments/files/25724127/server-audit.log)
[server-build.log](https://github.com/user-attachments/files/25724125/server-build.log)
[server-chat.log](https://github.com/user-attachments/files/25724124/server-chat.log)

**Comment by veldetf** [comment]
I see a similar issue in that you cannot add an item (single nor stack) to a crate attached to a boat while sitting in the boat.  Since it is not possible to hold shift without exiting the boat, I open inventory while attempting to add an item to a crate. When you attempt to add an item to a crate, instead it removes an item.

When you are not sitting in the boat, you can add a stack to a crate attached to a boat if your inventory is open, or if a chest is open.  You cannot add a stack to a crate attached to a boat otherwise.

When you hover the mouse over a crate attached to a boat that has contents, the tooltip does not describe its contents.

Expected behavior:  a crate on a boat should act like a crate on the ground.

v1.22.3
